src/services/point.service.ts

```diff
@@ -15,6 +15,10 @@ export class PointService {
       return;
     }
 
+    if (new Date().getDay() === 2) {
+      amount = amount * 2;
+    }
```

Comment:
```suggestion
   // give bonus points on tuesdays
    if (new Date().getDay() === 2) {
      amount *= 2;
    }
```

Reply:
Is day zero based or one based? Does it start with Sunday or Monday?

Reply:
> 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getDay
It starts with Sunday = 0

Reply:
we not doubling the amount for gambling